DOL.WHD.Section14c.Web/src/v4/hello-world.component.ts

```diff
@@ -5,6 +5,10 @@ import { Component } from '@angular/core';
   template: `
     <h1>Hello, World!</h1>
     <p>This is an Angular 4 component :)</p>
+    <p>This next thing is an upgraded v1 component:</p>
+    <div style="background-color:#eee; padding:0 1rem;">
+      <main-top-nav></main-top-nav>
+    </div>
   `
 })
 export class HelloWorldComponent {}
```

Comment:
Uses the Angular `main-top-nav` directive, which is an upgraded AngularJS component.

Reply:
🤘